src/Orchard.Web/Modules/Orchard.DynamicForms/Views/Elements/ReCaptcha.cshtml

```diff
@@ -5,7 +5,7 @@
     var element = (ReCaptcha)Model.Element;
     var publicKey = (string)Model.PublicKey;
 }
-@if (!String.IsNullOrWhiteSpace(publicKey)) {
+@if (HasText(publicKey)) {
```

Comment:
I wonder why you're using `HasText` on strings? Isn't that intended to be used on objects other than strings?